spec/app/domain/util/open_ssl/x509/shared_context.rb

```diff
@@ -0,0 +1,58 @@
+$LOAD_PATH << File.expand_path('../../../../../../..', __FILE__)
```

Comment:
Wow this is awful. A necessary evil for decoupling specs from Rails I guess?

Reply:
Yes it's awful.  And yes that's one reason, but it's not necessary.  I'll figure out a better way that will avoid this, but don't want to hold up the PR for it...